### DISCUSSION TITLE
Need help

### DISCUSSION THREAD

**Discussion by avindra99** [braintree/braintree_java#123] [Q&A]
How can I run the whole project and where can I find the document about the project and implementation

**Comment by kmichalak** [comment]
I'm quite unsure what you mean by "run the project". It is a library that should be used in your application to simply perform some API calls, nothing more. Can you share some more info what you want to achieve?

**Comment by avindra99** [comment]
Hi Kmichalak, thanks for your response, I am new to this open source project, I am in the final semester of my graduation. And I am very interested in payment API's. To start contributing I need some understanding on this project, where can I find documentation for this project. And also can you please let me know the issues, bugs or new enhancements to work on ?

**Comment by kmichalak** [comment]
You can find Braintree Developer Docs at https://developer.paypal.com/braintree/docs/
I sugges starting with an overview documentation available at https://developer.paypal.com/braintree/docs/start/overview 
All the issues can be found in the corresponding repository for each SDK version (Java, JavaScript, Ruby, Python, etc.).
